client/web/src/enterprise/batches/workspaces-list/Icons.tsx

```diff
@@ -14,6 +17,19 @@ export const CachedIcon: React.FunctionComponent<React.PropsWithChildren<unknown
     />
 )
 
+export const PartiallyCachedIcon: React.FunctionComponent<React.PropsWithChildren<{ count: number }>> = ({ count }) => {
+    const label = `A partial cache result was found for ${count} ${pluralize('step', count)} in this workspace.`
```

Comment:
This feels a little unclear: are the step caches themselves partial, or is the partial aspect referring to the batch change as a whole? (I mean, I'm personally about 98% sure I know the answer, but I'm also not the average user.)

Do we need the word partial in here at all? What about:

```suggestion
    const label = `This workspace contains cached results for ${count} ${pluralize('step', count)}.`
```

Reply:
Good suggestion, I'll take that! My wording skills are poor :)

Reply:
I actually would have preferred `${count} steps are cached in this workspace.` to be more active, but then we have to inflect `is/are` as well, and fuck that.

Reply:
That's the spirit.